notebooks/ICESat-2_Cloud_Access/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -53,7 +53,7 @@
     "### **Example of end product** \n",
```

Comment:
<p>This returns:</p><p> </p><pre class="ql-syntax" spellcheck="false">[None, None, None, None]
</pre><p> </p><p>After the nicely rendered metadata. I know the next release will have a fix for the 0 size issue, so I'm not sure if that's what is supposed to be shown here. Either way we may want to describe what this means. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I don't think it refers to the size of the files, because it still appears when using the updated version of earthaccess with the fix for the 0 size issue. I am not sure what` [None, None, None, None]` means when using `[display(r) for r in results]`. Any suggestions @betolink ?

Reply:
`[None, None, None, None]` are the returns from displaying an object in the jupyter notebook, they don't indicate a value of the metadata or anything, they don't show up if we just eval a single granule on a cell.

Reply:
Have added a one sentence explanation of this to the notebook